frontends/verilog/verilog_lexer.l

```diff
@@ -41,78 +48,113 @@
 #pragma clang diagnostic ignored "-Wmisleading-indentation"
 #endif
 
-#include "kernel/log.h"
-#include "frontends/verilog/verilog_frontend.h"
+#include "frontends/verilog/verilog_lexer.h"
 #include "frontends/ast/ast.h"
-#include "verilog_parser.tab.hh"
+#include "frontends/verilog/verilog_location.h"
+#include "kernel/log.h"
+#include <vector>
+#include <memory>
 
 USING_YOSYS_NAMESPACE
 using namespace AST;
 using namespace VERILOG_FRONTEND;
-
-#define YYSTYPE FRONTEND_VERILOG_YYSTYPE
-#define YYLTYPE FRONTEND_VERILOG_YYLTYPE
+using parser = frontend_verilog_yy::parser;
+//#define YYSTYPE FRONTEND_VERILOG_YYSTYPE
+//#define YYLTYPE FRONTEND_VERILOG_YYLTYPE
 
 YOSYS_NAMESPACE_BEGIN
-namespace VERILOG_FRONTEND {
-	std::vector<std::string> fn_stack;
-	std::vector<int> ln_stack;
-	YYLTYPE real_location;
-	YYLTYPE old_location;
-}
+#undef YY_DECL
+#define YY_DECL parser::symbol_type VerilogLexer::nextToken()
+
+#undef yyterminate
+#define yyterminate() terminate()
```

Comment:
I'm not sure exactly how the warning relates to this line, but it should probably be checked.